_examples/blockchain.adoc

```diff
@@ -0,0 +1,107 @@
+= Blockchain Analysis Notebook
+:page-link: blockchain
+:page-weight: 101
+:page-labels: [Python, Jupyter, spark-notebook]
+:page-layout: application
+:page-menu_template: menu_tutorial_application.html
+:page-description: The blockchain notebooks are examples of how to explore graph data using GraphX and GraphFrames on top of OpenShift using Spark. It uses the real Bitcoin blockchain data to create transaction graph for the analysis.
+:page-project_links: ["https://github.com/Jiri-Kremser/bitcoin-insights"]
+
+[[introduction]]
+== Introduction
+
+Cryptocurrencies attract various groups of people. Among other it could be
+investors, people from retail, tech enthusiasts, crypto-anarchists, etc.
+We are not going to focus on anything else than the raw technology behind the
+Blockchain, leaving aside all the ideology and hype that comes with the Bitcoin.
+
+The https://spark.apache.org/graphx/[GraphX] based notebook uses the https://github.com/spark-notebook/spark-notebook[spark-notebook] as the notebook technology,
+while the second one uses https://graphframes.github.io/[GraphFrames] and Jupyter notebook. Also the second
```

Comment:
i think i would follow your pattern from the earlier in the sentence, so something like:

```
while the https://graphframes.github.io/[GraphFrames] based notebook uses project Jupyter.
```